flink-connectors/flink-sql-connector-elasticsearch7/pom.xml

```diff
@@ -135,10 +131,6 @@ under the License.
 									<pattern>org.elasticsearch</pattern>
 									<shadedPattern>org.apache.flink.elasticsearch7.shaded.org.elasticsearch</shadedPattern>
 								</relocation>
-								<relocation>
```

Comment:
Do we need to remove this one? If yes, how about es6?

Reply:
Yes, sql-connector-es6 has been fixed this in https://issues.apache.org/jira/browse/FLINK-16287.
BTW, sql-connector-es6 is easier to found because  end2end test in sql_client_test cover, so this PR add e2e test for sql-connector-es7 too.